site/content/docs/5.1/components/navs-tabs.md

```diff
@@ -172,6 +172,62 @@ Nav tabs light only differ visually, with a full width bottom border and a diffe
 {{< /example >}}
 <!-- End mod -->
 
+### Nested tabs
+
+
+Nav tabs light is nested in a tab for adding a level of depth in information organization.
+
+{{< example >}}
+<div role="tablist" aria-owns="nav-tab1 nav-tab2 nav-tab3 nav-tab4">
+  <ul class="nav nav-tabs" id="nav-tab-with-nested-tabs">
+    <li class="nav-item">
+      <button class="nav-link active" aria-current="page" id="nav-tab1" data-bs-toggle="tab" data-bs-target="#tab1-content" type="button" role="tab" aria-controls="tab1-content" aria-selected="true">Tab 1</button>
```

Comment:
Why are we using `<button>`s like in all other examples in this page? Maybe we already talk about it but I don't remember :innocent: 
It causes a problem: it is possible to focus the element with the keyboard when it is disabled.

Reply:
Replacing `<button>` by `<a>` doesn't seem to do the job... except if I replace only for the disabled item...